engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -256,6 +258,11 @@ private void initRenderGraph() {
         renderGraph.addNode(toneMappingNode, "toneMappingNode");
         renderGraph.addNode(bloomPassesNode, "bloomPassesNode");
         renderGraph.addNode(blurPassesNode, "blurPassesNode");
+        if (renderingConfig.isVrSupport()) {
+            Node copyToVRFrameBufferNode = nodeFactory.createInstance(CopyToVRFrameBuffersNode.class);
+            ((CopyToVRFrameBuffersNode) copyToVRFrameBufferNode).setOpenVRProvider(this.vrProvider);
```

Comment:
Not so much Java related as about our architecture with `Context` serving as a sort of object store (in a less global way than the `CoreRegistry that it is replacing). 

I suspect what Manu is getting at here is that if it was added to a Context it could be made available in other places later on, for instance if a module wanted to add some VR related effect it could eventually `@In OpenVRProvider vrProvider;` then use it for stuff.

That's a guess though :-)